### PR TITLE
Feature/rotation metadata

### PR DESCRIPTION
**Pull request by miguel-lorenzo**

### CODE REVIEW

File: tests/test_parsing.py [hunk 1]
```diff
@@ -113,8 +113,59 @@ def test_get_correct_fps2():
...
+            minor_version   : 0
+            compatible_brands: isommp42
+            com.android.version: 10
+            encoder         : Lavf58.29.100
+            Stream #0:0(eng): Video: rawvideo (RGB[24] / 0x18424752), rgb24, 480x720 [SAR 1:1 DAR 2:3], q=2-31, 995328 kb/s, 120 fps, 120 tbn, 120 tbc (default)
+        """
+    )
+
+    info = parse_ffmpeg_header(sample)
+    assert info["rotate"] == 270
```
Comment (almarklein) [inline tests/test_parsing.py:163]: `== "270"` ;)

Comment (almarklein) [reply tests/test_parsing.py:163]: No wait. We should probably cast it to a float (in a try-except) in `_parsing.py`.

Comment (almarklein) [reply tests/test_parsing.py:163]: @miguel-lorenzo any chance you could cast to a float or int in `_parsing.py`? Then the tests should pass :)

Comment (miguel-lorenzo) [reply tests/test_parsing.py:163]: Done!

File: imageio_ffmpeg/_parsing.py [hunk 2]
```diff
@@ -165,6 +165,14 @@ def parse_ffmpeg_header(text):
             )
         )
 
+    # get the rotate metadata
+    reo_rotate = re.compile('rotate\s+:\s([0-9]+)')
```
Comment (almarklein) [inline imageio_ffmpeg/_parsing.py:169]:
```suggestion
    reo_rotate = re.compile("rotate\s+:\s([0-9]+)")
```

**Comment by almarklein** [comment]
Thanks!

**Pull request merged**